vhdbuilder/packer/install-dependencies.sh

```diff
@@ -278,6 +278,18 @@ string_replace() {
   echo ${1//\*/$2}
 }
 
+# Limit number of parallel pulls to 2 less than number of processor cores in order to prevent issues with network, CPU, and disk resources
+# Account for possibility that number of cores is 3 or less
+num_proc=$(nproc)
+if [[ $num_proc -gt 3 ]]; then
```

Comment:
Do you want this for >= 3 or just > 3? I assume the former - so you can switch to `ge` instead of `gt`

Reply:
If the number of processors is 3, it would result in 1, which is the same as the else statement. It would not make a difference either way that I can tell. Do you want me to change it?

Reply:
Fair, doesn't matter then